README.md

```diff
@@ -18,6 +18,9 @@ Returns `true` if you are calling the function from the renderer process.
 - **is.main()**  
 Returns `true` if you are calling the function from the main process.
 
+- **is.standarBrowser()**  
+Return `true` if you are calling the function from a standar browser.
```

Comment:
Since this is an edge case, please write a comment that explains why this function is needed.

Also fix the typos ;)